bot.py

```diff
@@ -467,7 +466,7 @@ async def tournament(ctx, *args):
             await updateTournamentList()
 
 @bot.command()
-@commands.check(isStaff)
+@commands.has_any_role(ROLE_STAFF, ROLE_VIP)
```

Comment:
What is the point of this? It seems like it would make the staff-checking functionality of the code harder to change in the future. If all of the checking is done in `inStaff`, then adding future staff-like roles in the future won't require changing all of these lines to accommodate the new role(s).

Reply:
Yeah, it's not that practical. Pushed a commit restoring it back to the original.